Dockerfile

```diff
@@ -141,7 +141,8 @@ RUN <<EOF
         -a slirp4netns \
         -a rootlesskit \
         -a rootlesskit-docker-proxy \
-        -a nsenter
+        -a nsenter \
+        --disable-manifest-validation
```

Comment:
This line should be removed when schema is merged and we've updated to valid SDK